### DISCUSSION TITLE
Express Checkout - translation to portuguese missing (Or continue below)

### DISCUSSION THREAD

**Discussion by modicharm** [woocommerce/woocommerce#48539] [Checkout Flow]
Hello!

Express Checkout, the translation to portuguese of the expression (Or continue below) is missing.

Thank you!

**Comment by nielslange** [comment]
Hello @modicharm 👋

I noticed that the string "Or continue below" on [this page](https://translate.wordpress.org/projects/wp-plugins/woocommerce/stable/pt/default/?filters%5Bterm%5D=Or+continue+below&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc) hasn't been translated yet. Since translations are managed by community members like yourself, you can add the missing translation there. For more details on how to submit translations, please refer to the [Translation Handbook](https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org).

I'm closing this thread as there isn’t anything actionable for us on this issue. That said, if you have any other questions or suggestions, feel free to create a new issue or thread.
